Vagrantfile

```diff
@@ -2,65 +2,55 @@
 # vi: set ft=ruby :
 VAGRANTFILE_API_VERSION = "2"
 
-ALLINONE = (ENV['ALLINONE'] || 0).to_i
+PROD = (ENV['PROD'] || 0).to_i
```

Comment:
It's for vagrant. Initially, we were having ``ALLINONE`` and multinode setup, but it seems that openshift and scanner do not run well together. So, I moved from a single node setup to 2 node setup, for development, and a multinode setup with 3-4 nodes for a prod like setup.

My choice of using ``PROD`` is mostly because of lack of good words.